gridss.wdl

```diff
@@ -33,6 +33,7 @@ task GRIDSS {
         BwaIndex reference
         String outputPrefix = "gridss"
 
+        Int jvmHeapSizeGb = 1
```

Comment:
You can set this to 30 GB so it will be usable in production.
During testing you can override this in the settings file.